samples/ots16/event.json

```diff
@@ -20,38 +20,6 @@
     "organizer_name": "FOSSASIA",
     "privacy": "public",
     "schedule_published_on": "2016-05-04 09:00:00",
-    "social_links": [
-        {
```

Comment:
They seemed redundant as Social links were already in social_links.json, where they are actually created from. OK, I'll reset commit.